src/Image.js

```diff
@@ -27,28 +27,28 @@ const propTypes = {
 };
 
 const defaultProps = {
-  responsive: false,
+  fluid: false,
   rounded: false,
-  circle: false,
+  roundedCircle: false,
   thumbnail: false
 };
 
 class Image extends React.Component {
   render() {
     const {
-      responsive,
+      fluid,
       rounded,
-      circle,
+      roundedCircle,
       thumbnail,
       className,
       ...props
     } = this.props;
     const [bsProps, elementProps] = splitBsProps(props);
 
     const classes = {
-      [prefix(bsProps, 'responsive')]: responsive,
-      [prefix(bsProps, 'rounded')]: rounded,
-      [prefix(bsProps, 'circle')]: circle,
+      [prefix(bsProps, 'fluid')]: fluid,
+      rounded,
+      'rounded-circle': roundedCircle,
```

Comment:
I wonder if i's better to remove the `rounded` prop completely and point people to https://getbootstrap.com/docs/4.0/utilities/borders/ in the docs, which is what upstream is doing.

what do you think @taion ?